source/features/remove-label-faster.css

```diff
@@ -16,7 +16,7 @@
 
 .rgh-remove-label-faster:is(:focus, :hover) svg {
 	background-color: currentColor;
-	fill: var(--rgh-remove-label-faster-color);
+	fill: hsl(0deg, 0%, calc((1 - var(--lightness-switch)) * 100%));
```

Comment:
Thank you for the PR! Can you check if there’s a GitHub-native variable we can use instead of HSL? They have a lot of dynamic variables that I’m sure are useful for  this

Reply:
Can you find out why the previous piece of code is now undefined? I think it’s best to keep using that since it used to work best (reusing the actual color of the label instead of just black/white).